.github/workflows/ci.yml

```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Linter
         run: poetry run flake8
 
+      - name: Test
+        run: poetry run nosetests
```

Comment:
> **Note to Users**
>
> Nose has been in maintenance mode for the past several years and will likely cease without a new person/team to take over maintainership. New projects should consider using Nose2, py.test, or just plain unittest/unittest2.

Sollen wir nicht vor allem bei neue Projekte auf pytest setzen?

https://docs.pytest.org/en/stable/

Nose wird nicht mehr weiterentwickelt und nose2 scheint sehr eingeschränkt zu sein - eine schlanke Alternative zu pytest, weil in der Vergangenheit wurde oft gesagt, dass pytest zu magisch ist, aber mittlerweile denke ich es ist zu Standard geworden und ist richtig stabil.